KachnaOnline.Business.Data/Repositories/EventsRepository.cs

```diff
@@ -44,5 +50,10 @@ public IAsyncEnumerable<Event> GetStartingBetween(DateTime from, DateTime to)
                 .Where(e => e.From >= from && e.From <= to)
                 .AsAsyncEnumerable();
         }
+
+        public async Task<Event> GetWithLinkedStates(int eventId)
+        {
+            return await Set.Include(e => e.LinkedPlannedStates).FirstOrDefaultAsync(e => e.Id == eventId);
+        }
```

Comment:
You explicitly included the planned states collection in all _Get_ requests, there's no point in having another method that does exactly the same thing.

Reply:
This is what I wanted to talk to you about. I believe this is generally useful for any requests asking for a specific event that I would like to add at least linked planned states IDs to all of these responses. Would you agree?

Reply:
Removed. If necessary to reverse, it can be done, too.